src/patternfly/components/InputGroup/styles.scss

```diff
@@ -11,16 +11,6 @@
   --pf-c-input-group__text--BorderSize: var(--pf-global--BorderWidth--sm);
   --pf-c-input-group__text--BorderColor: var(--pf-global--BorderColor--dark);
   --pf-c-input-group__text--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
-  --pf-c-input-group__action--FontSize: var(--pf-global--FontSize--md);
-  --pf-c-input-group__action--Color: var(--pf-global--Color--200);
-  --pf-c-input-group__action--PaddingTop: var(--pf-global--spacer--sm);
-  --pf-c-input-group__action--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-input-group__action--PaddingBottom: var(--pf-global--spacer--xs);
-  --pf-c-input-group__action--PaddingLeft: var(--pf-global--spacer--sm);
-  --pf-c-input-group__action-hover--Color: var(--pf-global--Color--100);
-  --pf-c-input-group__action-focus--Color: var(--pf-global--Color--100);
-  --pf-c-input-group__action-form-control--PaddingRight: var(--pf-global--spacer--xl);
-  --pf-c-input-group__action-form-control--PaddingLeft: var(--pf-global--spacer--xl);
   --pf-c-input-group__textarea--MinHeight: var(--pf-global--spacer--xl);
   --pf-c-input-group__button--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-input-group__button--PaddingLeft: var(--pf-global--spacer--md);
```

Comment:
L35 (`align-items: stretch;`) can be removed since `stretch` is the default and that isn't overwriting anything.